pkg/bff/api/v1/api.go

```diff
@@ -69,6 +71,22 @@ type MemberDetails struct {
 	Certificate map[string]interface{} `json:"certificate"`
 }
 
+// AnnouncementsReply contains up to the last 10 network announcements that were made in
+// the past month. It does not require pagination since only relevant results are returned.
+type AnnouncementsReply struct {
+	Announcements []*Announcement `json:"announcements"`
+	LastUpdated   string          `json:"last_updated,omitempty"`
+}
+
+// Announcement represents a single network announcementthat can be posted to the
+// endpoint or returned in the announcements reply.
+type Announcement struct {
+	Title    string `json:"title"`
+	Body     string `json:"body"`
+	PostDate string `json:"post_date,omitempty"` // Ignored on POST only available on GET
+	Author   string `json:"author,omitempty"`    // Ignored on POST only available on GET
+}
+
```

Comment:
This is serving as the database model as a shortcut that couples the db to the RPC handler.